go-oapi-codegen/pom.xml

```diff
@@ -121,7 +121,7 @@
     </dependencies>
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <openapi-generator-version>7.9.0</openapi-generator-version>
+        <openapi-generator-version>6.0.0</openapi-generator-version>
```

Comment:
Should be 6.6.0

Reply:
I notice. It is what it used to be and 6.6.0 breaks the `mvn package`